packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1512,6 +1573,10 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
   ) {
     deadline = dl;
 
+    if (enableProfileModeMetrics) {
+      resumeActualRenderTimerIfPaused();
+    }
+
```

Comment:
Can you move the user timing block above findHighestPriorityRoot so that it is next to this one? Since they do the same thing, it's nice to have them colocated.

Reply:
Moving that one up would break things, since `findHighestPriorityRoot()` sets `nextFlushedExpirationTime`. I can move this one down though.